scenario/deploy.xml

```diff
@@ -6,6 +6,7 @@
  */
 -->
 <scenario xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../config/scenario.xsd">
+    <step name="check-evn-variable-mage-error-report-dir-nesting-level" type="Magento\MagentoCloud\Process\Deploy\CheckEnvVarMageErrorReportDirNestingLevel"/>
```

Comment:
For me, looks like the class name contains too many words. I would change to something like this

```
<step name="check-env-nesting-level" type="Magento\MagentoCloud\Process\Deploy\CheckEnvNestingLevel"/>
```

Reply:
Done